### PR TITLE
IDEM-52

### PR DESCRIPTION
**Pull request by xiangzhe0812**
Screen Shot 2022-08-30 at 10 26 42 pm

### CODE REVIEW

File: src/services/EmailService.ts [hunk 1]
```diff
@@ -52,13 +55,40 @@ export class EmailService implements IEmailService {
...
+        const privateKeyArmored = fs.readFileSync(
+            path.join(__dirname, "../../") + "/test_idem_com_au.asc",
+            { encoding: "utf8" }
+        );
+
+        const privateKeys = await openpgp.readPrivateKeys({
+            armoredKeys: privateKeyArmored
+        });
+
+        const passphrase = process.env.PGP_PASSPHRASE;
```
Comment (brettvniekerk) [inline src/services/EmailService.ts:71]: `as string` ?

File: src/services/EmailService.ts [hunk 2]
```diff
@@ -52,13 +55,40 @@ export class EmailService implements IEmailService {
...
+            privateKey: privateKeys[0],
+            passphrase
+        });
+
+        const cleartextMessage = await openpgp.sign({
+            message: unsignedMessage,
+            signingKeys: privateKey
+        });
+
+        console.log(cleartextMessage);
```
Comment (brettvniekerk) [inline src/services/EmailService.ts:83]: is this meant to be left over? is there not a logging suite?

**Review by brettvniekerk — DISMISSED** [review]
LGTM once comments resolved

File: src/services/EmailService.ts (hunk 1 above)
Comment (pondpiu) [inline src/services/EmailService.ts:71]: get env from Config?

**Review by bitcoinbrisbane — APPROVED** [review]

File: src/services/EmailService.ts (hunk 2 above)
Comment (bitcoinbrisbane) [reply src/services/EmailService.ts]: @xiangzhe0812 lets remove.  Good catch @brettvniekerk

**Review by JasminFarhi — APPROVED** [review]

**Pull request merged**